appveyor.yml

```diff
@@ -1,27 +1,106 @@
 version: 1.0.{build}
 
+# Skipping commits affecting these files
+skip_commits:
+   files:
+    - '**/*.md'
+    - '**/*.png'
+    - '**/*.jpg'
+
+# setting up etc\hosts file
+# XXX - use this instead of batch file generated one
+hosts:
+  localhost: 127.0.0.1
+#  localhost: ::1 # appveyor doesn't like "duplicates"
+  mpa.one.microsoft.com: 127.0.0.1
+  google-public-dns-a.google.com: 8.8.8.8
+  www.no-such-host.com: 10.0.0.20
+
 environment:
-    matrix:
-      - BUILDER: msvc
-        CPU:     x86
-      - BUILDER: msvc
-        CPU:     x64
-      - BUILDER: mingw
-        CPU:     x86
-    # - BUILDER: mingw
-    #   CPU:     x64
+  global:
+    # CFLAGS: '-Wall --coverage'
+    # LDFLAGS: '-lgcov --coverage'
+    CL: /nologo /MP
+  matrix:
+    - BUILDER: msvc
+      CPU:     x86
+    - BUILDER: msvc
+      CPU:     x64
+    - BUILDER: mingw
+      CPU:     x86
+      MSYSTEM: MINGW32
+    - BUILDER: mingw
+      CPU:     x64
```

Comment:
Good to see 64-bit MinGW working with AppVeyor. But CygWin fails miserably 🤕. Oh well. I'll merge this later when I have the time. Cheers.

Reply:
hehe miserably indeed. 👍